website/blog/2018-11-01-oss-roadmap.md

```diff
@@ -0,0 +1,67 @@
+* Support 64-bit libraries on Android.
+* Enable debugging under the new architecture.
+* Improve support for CocoaPods, Gradle, Maven, and new Xcode build system.
+
+## ✅ Testing Infrastructure
+
+When Facebook engineers publish code, it's considered safe to land if it passes all tests. These tests identify whether a change might break one of our own React Native surfaces. Yet, there are differences in how Facebook uses React Native. This has allowed us to unknowingly break React Native in open source.
+
+We'll shore up our internal tests to ensure they run in an environment that is as close as possible to open source. This will help prevent code that breaks these tests from making it to open source. We will also work on infrastructure to enable better testing of the core repo on GitHub, enabling future pull requests to easily include tests.
+
+Combined with the reduced surface area, this will allow contributors to merge pull requests quicker, with confidence.
+
+## 📜 Public API
+
+Facebook will consume React Native via the public API, the same way open source does, to reduce unintentional breaking changes. We have started converting internal call sites to address this. Our goal is to converge on a stable, public API, leading to the adoption of semantic versioning in version 1.0.
+
+## 📣 Communication
+
+React Native is one of the [top open source projects on GitHub](https://octoverse.github.com/projects) by contributor count. That makes us really happy, and we'd like to keep it going. We'll continue working on initiatives that lead to involved contributors, such as increased transparency and open discussion. The documentation is one of the first things someone new to React Native will encounter, yet it has not been a priority. We'd like to fix that, starting with bringing back auto-generated API reference docs, and creating additional content focused on creating [quality user experiences](http://facebook.github.io/react-native/docs/improvingux).
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">So happy we could get a bunch of React Native core contributors together for dinner at React Conf 2018! <a href="https://t.co/p5tEWhGAwy">pic.twitter.com/p5tEWhGAwy</a></p>&mdash; Eli White (@Eli_White) <a href="https://twitter.com/Eli_White/status/1056008011120955392?ref_src=twsrc%5Etfw">October 27, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Timeline
+
+We're planning to land these projects throughout the next year or so. Some of these efforts are already ongoing, such as [JSI which has already landed in open source](https://github.com/facebook/react-native/compare/e337bcafb0b017311c37f2dbc24e5a757af0a205...8427f64e06456f171f9df0316c6ca40613de7a20). Others will take a bit longer to complete, such as reducing the surface area. We'll do our best to keep the community up to date with our progress.
```

Comment:
mention that the JSI will be explained more in a blogpost coming "soon"? And that in the meantime there is the talk by Ram at ReactConf that explains it -> https://www.youtube.com/watch?v=UcqRXTriUVI ?